buildSrc/src/main/kotlin/Project.kt

```diff
@@ -99,7 +99,7 @@ object Versions {
   const val googlePlayServicesLocation = "18.0.0"
   const val googleCarAppLibrary= "1.1.0-alpha01"
   const val kotlinCoroutines = "1.3.9"
-  const val junit = "4.12"
+  const val junit = "4.13"
```

Comment:
why bumping? do we need some functionality from it?

Reply:
From version 4.13 is available `assertThrows` from the box.